src/main/windows.ts

```diff
@@ -54,6 +54,19 @@ export function createMainWindow(): Electron.BrowserWindow {
     browserWindow = null;
   });
 
+  // Focusing on new window loads the possibly clicked "Show Me" fiddle
+  browserWindow.on('focus', () => {
+    browserWindow?.webContents.send(IpcEvents.MENU_IPC_RENDER);
```

Comment:
Can we use the `IpcMainManager` and `IpcRendererManager` modules to handle IPC events?